R/lookup.R

```diff
@@ -139,7 +139,7 @@ p4b_scenarios_lookup <- c("target_b2ds", "target_cps", "target_rts",
                           "target_sds")
 
 # allowed input years p4b production data file
-p4b_production_data_years_lookup <- c(2020, 2021)
+p4b_production_data_years_lookup <- c(2019, 2020)
```

Comment:
just to go sure cfg$AnalysisPeriod$Years.Startyear, which used to determin the start_year was 2021 in the ST_TESTING projects before we manually corrected it. Just making sure we do not overlook anything in this regard?

Reply:
for loans, it is currently not possible to have start year 2021. it's always either 2019 or 2020, based on the scenario data start years that they have. I previously assumed identical handling as in P4I, but this is not the case. so yes, for now this is correct

Reply:
completely removed this for now (only 2020 supported hard coded for now)